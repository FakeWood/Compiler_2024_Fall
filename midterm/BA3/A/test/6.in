f(g(h(1546.4154, -1578.598, 305.01111111), 99789621))