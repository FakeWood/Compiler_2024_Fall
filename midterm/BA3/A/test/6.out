399261160.978